### ISSUE TITLE
Event signature check fails in Python 3.6.2

### ISSUE THREAD

**Issue by dperetti** [stripe/stripe-python#346]
Apparently the issue is here:
https://github.com/stripe/stripe-python/blob/master/stripe/webhook.py#L59

I've been able to make it work with this change:

```python
        # signed_payload = "%d.%s" % (timestamp, payload)  # Doesn't work
        signed_payload = "%d.%s" % (timestamp, payload.decode('utf8'))  # works !
```

(payload is bytecode so `str(payload)` creates a representation of the bytecode (using stuff like `b\\` prefix))

**Comment by ob-stripe** [comment]
Hey @dperetti, I pushed a fix for this issue yesterday in #345.

Can you try updating the stripe module to the latest version (1.65.1) and check if the issue still happens?

**Comment by dperetti** [comment]
Aha you're right I was not using the very latest ! :-)
